### ISSUE TITLE
Wrong output generated for nested modules

### ISSUE THREAD

**Issue by lukaszreszetow** [vanniktech/gradle-dependency-graph-generator-plugin#112]
Hi, it seems like there is an issue for nested modules. 

For example let's create module `sample` and then inside this module, we generate another one `sampleNested` module.

Now if we run `generateDependencyGraph` for all modules, for `sample` module inside build/reports folder, we receive the same output, like the one that is generated for `sampleNested`. 

In the tree, the top most module is `sampleNested` and not `sample`.

I can create sample app, if my explanation is to.. idle :D

**Comment by vanniktech** [comment]
Can you try the latest 0.6.0-SNAPSHOT?

**Comment by lukaszreszetow** [comment]
I was already using latest commit, and it's probably the same thing as 0.6.0-SNAPSHOT. Nevertheless I've tested with that, and the same thing happens.

**Comment by azabost** [comment]
Hi. I've just tested pretty much the same thing and I'm not even sure if it is no bug or just a little confusion about this particular use case.

Having this in the top-level `build.gradle.kts`:

```
// Allows running: ./gradlew generateDependencyGraph
apply(plugin = "com.vanniktech.dependency.graph.generator")

allprojects {
    // Applied to every project separately in order to allow running: ./gradlew :module:generateDependencyGraph
    apply(plugin = "com.vanniktech.dependency.graph.generator")
}
```

when you run a module-specific command, i.e. `./gradlew :moduleX:generateDependencyGraph`, the graph will not show the `moduleX` by default unless another module has a dependency on it. When it happens that another module has a dependency on `moduleX`, the graph won't show the arrows from `moduleX` to the other modules and this part is confusing. But I think there's nothing wrong with it because the whole point of that graph was to show the `moduleX` dependencies in the first place so the arrows would just point all the other nodes.

TL;DR it's probably alright

Let me know if you would like to get some sample reproduction with an explanation on the pictures.

**Comment by vanniktech** [comment]
I think this might be fixed by #116

**Comment by vanniktech** [comment]
I'm closing this issue due to inactivity. If you have any further input on the issue, don't hesitate to reopen this issue or post a new one.
